### DISCUSSION TITLE
Note repeat on the side pads when Keyboard mode is on

### DISCUSSION THREAD

**Discussion by soymonitus** [SynthstromAudible/DelugeFirmware#408] [Ideas]
We could use the two rows on the right, when keyboard mode is engaged, specially for Drum keyboard, to apply some note repeat. Have this as an example:
https://www.youtube.com/watch?v=2grLMVQ2b2s&ab_channel=VJFRANZK
By holding any of the 8 vertical pads we apply different rates for the note repeat, if you don't hold any of the 8 pads you play single notes as normal

**Comment by trappar** [comment]
Can you close this discussion since it's a duplicate of https://github.com/SynthstromAudible/DelugeFirmware/discussions/256 ?
